components/NavigationBar/examples/NavigationBarLayoutExample.m

```diff
@@ -19,11 +19,10 @@
 //
 //
 
+#import "MDFInternationalization.h"
```

Comment:
Move down to bottom of list to keep it alphabetized.

Reply:
Aren't all capital letter are in front of lower ones according to the ObjC style guide?

Reply:
Oh, possibly! Can you cite?

Reply:
I struggle to actually find it. Moved the import down :)

Reply:
Thank you!